css/styles.css

```diff
@@ -6,45 +6,56 @@
   box-sizing: border-box;
 }
 
+body {
+  background-color: lightslategrey;
+  margin: 0;
+  padding: 0;
+}
+
 .header {
-  border-radius: 0px 0px 10px 10px;
+  border-radius: 10px 10px 10px 10px;
   display: flex;
   justify-content: center;
-  background-color: aquamarine;
-  box-shadow: 5px 5px 5px grey;
+  background-color: white;
+  box-shadow: 4px 4px 4px 2px;
   /* position: fixed;
   top: 0;
   left: 0;
   width: 100%;*/
 }
 
 .card {
-  background-color: aqua;
-  border: solid;
-  border-radius: 5px;
-  padding: 10px;
-  margin: 10px;
-  box-shadow: 5px 5px burlywood;
+  background-color: white;
+  /*border: solid;*/
```

Comment:
please remove lines of code that are commented out. In general we write code for the moment, if you need something from before you can use git to travel back to prior commits.

Reply:
will do that! if I go back some commits can I download it and only take one single part of the code and bring it to the latest commit?